schema.sql

```diff
@@ -57,3 +57,9 @@ CREATE TABLE visits (
     date_of_visit date,
     PRIMARY KEY(id)
 );
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animal_id_asc ON visits(animal_id ASC);
+CREATE INDEX vet_id_asc ON visits(vet_id ASC);
+CREATE INDEX email_asc ON owners(email ASC);
```

Comment:
- [ ] The code is absolutely fine for adding indexes, but somehow two of queries are not improving. one's result is almost identical before and after, and the second one is actually got worse after indexing so there is something wrong. kindly consider approving these two of your queries.
you can apply this code for indexing 
```
   CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);
```
Also sometimes if you ran same query after indexing, it gets better after first tries, so you can try that also.
your last two queries screen shots before and after
before 
![qw](https://user-images.githubusercontent.com/96683843/236886239-36414b29-9cc7-4203-844a-0510876fd0f1.jpg)

after
![a](https://user-images.githubusercontent.com/96683843/236886264-787d669f-2f22-4078-9721-dfe4eb0b02af.jpg)

Reply:
@Zuhaib042 Thanks for the review! I noticed that after indexing, I even got worse query results in some cases. Could not figure out why. Had to drop the whole database to get it working!